content/docs/cucumber/api.md

```diff
@@ -680,6 +680,25 @@ Tags can be placed above the following Gherkin elements:
 * `Scenario Outline`
 * `Examples`
 
+In `Scenario Outline`, you can use tags on different example like below:
+
+```gherkin
+Scenario Outline: Steps will run conditionally if tagged
+  Given user is logged in
+  When user clicks <link>
+  Then user will be logged out
+
+  @mobile
+  Examples: 
+  | link |
```

Comment:
Lets indent the example table once as well, just so we're showing good behaviours

Reply:
@luke-hill fixed, thanks! 👍